src/components/Editor.vue

```diff
@@ -686,6 +687,23 @@ export default {
 				content.style.display = ''
 			}
 		},
+
+		/**
+		 * Helper method to debug serialization of content between markdown and HTML
+		 *
+		 * @param {object} editor The Tiptap editor
+		 */
+		debugContent(editor) {
+			const proseMirrorMarkdown = this.$syncService.options.serialize(editor.state.doc)
+			const markdownItHtml = markdownit.render(proseMirrorMarkdown)
+
+			logger.debug('markdown, serialized from editor state by prosemirror-markdown')
+			console.debug(proseMirrorMarkdown)
```

Comment:
Those console statements are not necessary, you could simply do it like
```js
logger.debug('markdown, serialized from editor state by prosemirror-markdown', { markdown: proseMirrorMarkdown })
logger.debug('HTML, serialized from markdown by markdown-it', { html: markdownItHtml })
logger.debug('HTML, as rendered in the browser by Tiptap', { html: editor.getHTML() })
```

Or you could even put it all together in one message:
```js
logger.debug('Current editor content', {
	html: editor.getHTML(),
	markdown: proseMirrorMarkdown,
	'markdown-it': markdownItHtml,
})
```

Reply:
When using `logger.debug` for logging the variables themselves, the variable value is folded, which makes it harder to read and compare while debugging. That's why I went with `console.debug()` for the variables themselves. Do you see a problem with my approach? Would you still prefer `logger.debug()`? :blush:

Reply:
No real problem, just personal preference (it looks cleaner to me and easier to copy only one message).
But if you like it this way more, that is fine :)